data/human/culture conversations.txt

```diff
@@ -622,3 +622,72 @@ mission "Sol-Bound Shuttle"
+			`	As he finishes, the ceiling lights flash back on. "Ah, well, there you go. Too bad I missed the celebrations, but that's the end of Earth Day and my night shift. I hope you had a good Earth Day, and have a good night!"`
+				decline
+
+			label tourist
+			`	You approach a woman with a bunch of oversized bags clustered around her seat. She's tapping rapidly on her datapad, but nonetheless she looks up when you approach. "Excuse me, Captain, I was just trying to figure out when the metro starts up again. Do you know?"`
+			choice
+				`	"I know just as much as you, sorry."`
+				`	"Maybe it's to do with the blackout?"`
+					goto "way in"
+			`	"Ha, they shut everything down for Earth Day and then assume all us offworlders get the idea. What do you know..."`
+			choice
+				`	"Earth Day?"`
+				`	"Shut everything down?"`
+			`	"Didn't you see what was happening outside on your way in? It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then, it was awfully primitive times.`
+				goto decided
+
+			label "way in"
+			`	"Yes, I suppose you must have seen what's happening outside on your way in. It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then, it was awfully primitive times.`
+
+			label decided
+			`	"So they decided that, on the 22nd of April each year, all of Earth would go into total blackout to symbolize that they didn't have enough energy to sustain themselves, and to remind them that the only way to advance from this unspectacular position was to keep pushing themselves forward, find new pathways to power, lest those blackouts became longer and more frequent. Fascinating, huh?"`
+			choice
+				`	"It seems a bit backwards."`
+				`	"It is fascinating, thanks. (Walk away.)"`
+					goto pass
+			`	"Well, perhaps. But Earth is like that sometimes. It's survived through so much, and without Earth Day to remind our ancestors that we must keep looking forward, maybe you wouldn't be in your fancy little spaceship, would you?" The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock, I've got to go now and see if the metro's opened so I can get to my accommodation. Safe travels!"`
+				decline
+
+			label pass
+			`	After the clock passes midnight, the lights in the spaceport come back on, followed by the signage and lights of the city surrounding it. A few workers come and snuff out the candles before putting them away in unseen closets in maintenance corridors.`
```

Comment:
```suggestion
			`	After the clock passes midnight, the lights in the spaceport come back on, followed by the signage and lights of the city surrounding it. A few workers come and snuff out the candles before putting them away in unseen closets.`
```
I think that extra bit kind of kills the final impact of bustling away this grand gesture and resuming normal course.